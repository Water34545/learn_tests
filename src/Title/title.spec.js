import React from 'react'
import Title from './title'

describe('Ешеду component', () => {
  it('should render Ешеду component with props', () =>{
    const component = shallow(<Title title='test '/>)
    expect(component).toMatchSnapshot()
  })
  it('should render Ешеду component without props', () =>{
    const component = shallow(<Title/>)
    expect(component).toMatchSnapshot()
  })
})