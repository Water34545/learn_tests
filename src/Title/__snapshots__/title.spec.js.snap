// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Ешеду component should render Ешеду component with props 1`] = `
<h1
  className="title"
>
  test 
</h1>
`;

exports[`Ешеду component should render Ешеду component without props 1`] = `
<h1
  className="title"
>
  Simple title
</h1>
`;
